### PR TITLE
chore(demo): add routerLinkActive on sidenav links

### PR DESCRIPTION
**Pull request by sirajc**
Highligh active link in SideNav, See preview below
image

### CODE REVIEW

**Comment by sirajc** [comment]
For active links increased the font-weight and used existing color `#373a3c` from _app.scss_

File: demo/src/style/app.scss [hunk 1]
```diff
@@ -89,6 +89,11 @@ p.github-buttons > iframe {
   cursor: pointer;
 }
 
+.bd-sidebar .nav > li > a.active{
+  font-weight: 700;
```
Comment (icfantv) [inline demo/src/style/app.scss]: how does the UI look w/ `bold` instead of a number?

**Comment by icfantv** [comment]
@sirajc, i just realized there's CSS already there for this.  IIRC, the `active` class just needs to be added to the `li` tag around the anchor.  [According to this](https://github.com/angular/angular/blob/master/modules/%40angular/router/src/directives/router_link_active.ts) it should just be a matter of moving the `routerLinkActive` to the `li` tag and removing the `[]`.

File: demo/src/style/app.scss [hunk 2]
```diff
@@ -89,6 +89,11 @@ p.github-buttons > iframe {
   cursor: pointer;
 }
 
+.bd-sidebar .nav > li > a.active{
```
Comment (wesleycho) [inline demo/src/style/app.scss]: Add a space before `{`

Comment (icfantv) [inline demo/src/style/app.scss]: that's not his code - that was mine originally.  also i neglected to put newlines on some compound css defs.

Comment (icfantv) [inline demo/src/style/app.scss]: gah, nevermind.  i missed that change.  i still need to put the newlines in...

**Comment by sirajc** [comment]
@icfantv Added `routerLinkActive` on `li` and removed CSS rule added by me
This is how it looks now
image

It added a good amount of margin, Also do we need to make it bold?

**Comment by icfantv** [comment]
Ooof, that looks horrible.  Please go ahead and:
1. remove the `.bd-toc-item .active` selector as this behavior is not wanted and isn't used elsewhere.
2. change the `font-weight` from 500 to 700 for the `.bd-sidebar .nav > .active > a` group of selectors.

Thanks.

**Comment by sirajc** [comment]
@icfantv This now looks like First screenshot in this PR (darker and bold without margins)

**Comment by icfantv** [comment]
Sweet!  If you're happy with it then LGTM.

**Pull request closed without merging**